packages/e2e-test-utils-playwright/src/editor/index.ts

```diff
@@ -89,4 +90,6 @@ export class Editor {
 		switchToLegacyCanvas.bind( this );
 	/** @borrows transformBlockTo as this.transformBlockTo */
 	transformBlockTo: typeof transformBlockTo = transformBlockTo.bind( this );
+	/** @borrows updatePost as this.updatePost */
+	updatePost: typeof updatePost = updatePost.bind( this );
```

Comment:
It is [recommended to avoid overloading](https://developer.wordpress.org/block-editor/contributors/code/testing-overview/e2e/#dont-overload-test-utils-inline-simple-utils) the end-to-end test utilities.

Let's make it a test-specific helper. Example - https://github.com/WordPress/gutenberg/blob/trunk/test/e2e/specs/editor/various/list-view.spec.js#L1052.

Reply:
Great 🙂 I just did that in [this commit](https://github.com/WordPress/gutenberg/pull/58676/commits/403e0f8d33345584454fd7b4932ddb0a730f3c04).

Reply:
TBH, I think `updatePost` should be an util because it's used so many times in different test suites 😅. Same as how `publishPost` and `saveDraft` are utils. This is not related to this PR though.

Reply:
I think the proper name for this utility would be `updatePublishedPost()`. The posts have different update flows based on their. We should account for them in `updatePost` or use more explicit names.

Reply:
Possibly we also want to consider saving multiple entities and different post types in the site editor as well. So I agree that this can be thought through in another PR/issue. 👍